src/commands/mod.rs

```diff
@@ -247,7 +248,23 @@ pub fn execute() -> Result<()> {
             .join(" ")
     );
 
-    run_command(&matches)
+    let _guard = sentry::init((
+        env!("SENTRY_DSN").to_string(),
```

Comment:
This should definitely be optional, as people may want to build it without enabling our SDK.

Reply:
yes, good suggestion!

I've made the whole `profiling` integration optional with a specific feature 👍🏻